Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># eclipse\r\nbin\r\n*.launch\r\n.settings\r\n.metadata\r\n.classpath\r\n.project\r\n\r\n# idea\r\nout\r\n*.ipr\r\n*.iws\r\n*.iml\r\n.idea\r\n\r\n# gradle\r\nbuild\r\n.gradle\r\n\r\n# other\r\neclipse\r\nrun\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 8c33b18dbb1a59930488aebbaad307301601f426)
+++ b/.gitignore	(date 1671547206176)
@@ -20,3 +20,5 @@
 # other
 eclipse
 run
+
+*.bat
\ No newline at end of file
Index: src/main/java/rawfish/artedprvt/item/itemFile/demo.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/rawfish/artedprvt/item/itemFile/demo.js b/src/main/java/rawfish/artedprvt/item/itemFile/demo.js
new file mode 100644
--- /dev/null	(date 1671546040819)
+++ b/src/main/java/rawfish/artedprvt/item/itemFile/demo.js	(date 1671546040819)
@@ -0,0 +1,1 @@
+status
\ No newline at end of file
